Aidan working branch + feats

aidanvancil opened the pull request.

aidanvancil committed:
Bunch of new feats... set desc

Email Access + Reset Password + Password Hashing + Login + Temporary Build folder that can be deleted soon. Email access changing still W.I.P.

aidanvancil committed:
Min password req + Reset Password

Both functions fully working. We still need to implement react html for it the be visually usable for users. But after this, we are done with signups/logins

NikhilSharma1234 commented on mezzo/server/utils/sendEmail.js:
```diff
@@ -0,0 +1,33 @@
+      if (error) {
+        console.log(error);
+      } else {
+        console.log('Email sent: ' + info.response);
```
We probably don't want console logs making it to our app.

NikhilSharma1234 reviewed (approved):
There are some console logs that we might not need, but otherwise looking solid.

aidanvancil committed:
Transport API calls?

Trying to figure out how to route from react to express, without having to constantly build.

aidanvancil committed:
Update sendEmail.js
